MovieLibrary/csvMovieList

```diff
@@ -0,0 +1,4 @@
+id,title,date

```

Comment:
ciekawostka: plik csv w niektórych standardach Europejskich (w tym Polska) jest rozdzielany średnikiem, ponieważ my Polacy kwoty rozdzielamy przecinkiem (zamiast kropką), co zaburza strukturę pliku

Źródło:
Semicolons are often used instead of commas in many European [locales](https://en.wikipedia.org/wiki/Locale_(computer_software)) in order to use the comma as the decimal separator and, possibly, the period as a decimal grouping character.
https://en.wikipedia.org/wiki/Comma-separated_values